packages/GA4Client/src/initialization.js

```diff
@@ -21,6 +23,13 @@ var initialization = {
     ) {
         mParticle._setIntegrationDelay(this.moduleId, true);
 
+        // Due to current limitations, the API for allowing a customer to cleanse
+        // their data before our cleansing occurs must be placed on the
+        // window.GoogleAnalytics4Kit object.  This exists when initializing MP
+        // SDK via snippet, but not via npm. If a customer uses npm, using the API
+        // requires window.GoogleAnalytics4Kit to exist on the page.
```

Comment:
The initial part is a little unclear. Does this make more sense?

```suggestion
        // To allow customers to cleanse their data before mParticle
        // internal cleansing occurs, we must add a simple API into the
        // window.GoogleAnalytics4Kit object. This object will be automatically
        // initialized when MP SDK initializes via snippet, but not via npm.
        // Customers using npm, a customer must manually add
        // window.GoogleAnalytics4Kit to the page themselves.
```